__tests__/app.test.js

```diff
@@ -212,3 +212,103 @@ describe("6. GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("7. POST /api/articles/:article_id/comments/", () => {
+  it("status:200, should responds with the new comment objects if user exists", () => {
+    const newPost = {
+      username: "butter_bridge",
+      body: "test body",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newPost)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: 1,
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
```

Comment:
As you know the exact details of the comment you are posting, you should be able to assert for specific properties on the response body.
`expect.objectContaining()` is still  valid though to allow for more properties in the future